src/System.Security.Cryptography.Cng/src/Internal/Cryptography/SymmetricImportExportExtensions.cs

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Text;
+using System.Diagnostics;
+using System.Globalization;
+using System.Security.Cryptography;
+
+namespace Internal.Cryptography
+{
+    internal static class SymmetricImportExportExtensions
+    {
+        public static CngKey ToCngKey(this byte[] key, string algorithm)
+        {
+            using (MemoryStream ms = new MemoryStream())
```

Comment:
Consider passing in the expected capacity here so that the MemoryStream doesn't need to continually grow its array.